WatchParty/client/selenium/movieResultTest.py

```diff
@@ -0,0 +1,53 @@
+from selenium import webdriver
+import unittest
+import time
+
+class ResolutionTest(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.driver = webdriver.Chrome(executable_path='/Users/Shahan/Desktop/chromedriver')
```

Comment:
I just noticed, this is a big oof! We need paths to be relative and not absolute! Also, the server will not be able to find this chromedriver! Get it into this branch as part of this commit, and instead change it to 
`executable_path='./chromedriver' `